grafana-plugin/e2e-tests/labels/createNewLabelKeysAndValues.test.ts

```diff
@@ -1,8 +1,15 @@
+import semver from 'semver';
+
 import { test, expect } from '../fixtures';
 import { clickButton, generateRandomValidLabel, openDropdown } from '../utils/forms';
 import { openCreateIntegrationModal } from '../utils/integrations';
 import { goToOnCallPage } from '../utils/navigation';
 
+test.skip(
+  () => semver.gt(process.env.CURRENT_GRAFANA_VERSION, '10.3.0'),
```

Comment:
_nit_ what about moving all mentions of `process.env.CURRENT_GRAFANA_VERSION` to `grafana-plugin/e2e-tests/utils/constants.ts`?

Reply:
Sounds good, will do it now